R/effectsize.R

```diff
@@ -132,17 +162,36 @@ effectsize.htest <- function(model, ...) {
 }
 
 #' @export
+#' @rdname effectsize
 #' @importFrom insight get_data get_parameters
 #' @importFrom bayestestR describe_posterior
-effectsize.BFBayesFactor <- function(model, ...){
+effectsize.BFBayesFactor <- function(model, type = NULL, ...){
   if (!requireNamespace("BayesFactor")) {
     stop("This function requires 'BayesFactor' to work. Please install it.")
   }
 
-  if (length(model) > 1)
+  if (length(model) > 1) {
     warning("Multiple models detected. Using first only.", call. = FALSE)
+    model <- model[1]
+  }
 
   if (inherits(model@numerator[[1]], "BFcontingencyTable")) {
+    if (is.null(type)) type <- "cramers_v"
+
+    f <- switch (tolower(type),
```

Comment:
My OCD is acting up every time I see `switch (` instead of `switch(` 😅

I will leave it up to you, but I'd just run `styler::style_dir()`.

Reply:
I style my code with `ctrl+i` and `ctrl+a` on the fly (which for some reason gives different styling than `styler::style_dir()`?)...